Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -97,7 +97,7 @@
   name: Mateba
   parent: [BaseWeaponRevolver, BaseMajorContraband]
   id: WeaponRevolverMateba
-  description: The iconic sidearm of the dreaded death squads. Uses .45 magnum ammo.
+  description: The iconic sidearm of the dreaded Death Squads, allegedly. Uses .45 magnum ammo.
```

Comment:
i think deathsquad might be one word with no space. also i think the comma should be a period

Reply:
May use ellipsis after deathsquad; keeps their existence ambiguous

Reply:
oh yeah an ellipsis would also work for sure